tracer/src/Datadog.Trace/Iast/Telemetry/ExecutedTelemetryHelper.cs

```diff
@@ -138,6 +145,6 @@ private string GetExecutedSinkTag(IastInstrumentedSinks vulnerability)
 
     private string? GetSourceTag(IastInstrumentedSources source)
     {
-        return SourceType.GetString((SourceTypeName)source);
+        return SourceType.GetString((SourceTypeName)source).Replace(".", "_");
```

Comment:
Rather than doing this `Replace()` on every request (generating a lot of garbage) it might be better to just have a separate `SourceType.GetAsTag(SourceTypeName)` method that returns the correct value.

You can always write a unit test to make sure they're funtionally equivalent to this 🙂

Reply:
Thanks for the suggestion! I have updated the code and added a unit test.